src/animatedRe.re

```diff
@@ -14,6 +14,32 @@ module CompositeAnimation = {
   [@bs.send] external reset : t => unit = "";
 };
 
+module Easing {
+  type t = float => float;
+
+  [@bs.module "react-native"] [@bs.scope "Easing"] external bounce : t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external circle : t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external cubic : t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external ease : t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external exp : t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external linear : t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external poly : t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external quad : t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external sin : t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external step0 : t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external step1 : t = "";
+
+  [@bs.module "react-native"] [@bs.scope "Easing"] external back : float => t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external elastic : float => t = "";
+
+  [@bs.module "react-native"] [@bs.scope "Easing"] external in_ : t => t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external inOut : t => t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external out : t => t = "";
+
+  [@bs.module "react-native"] [@bs.scope "Easing"]
+  external bezier: float => float => float => float => t = "";
```

Comment:
```
external bezier: (float, float, float, float) => t = "";
```

why not like this? 😉

Reply:
Hmm refmt should take care of that. Maybe run refmt on the files as well

Reply:
I can run refmt, but the entire file changes. Is that ok?

Reply:
looks good! thanks :)